shopinvader_api_address/schema/schema.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 ACSONE SA/NV
```

Comment:
should I put these schemas in module "shopinvader_schema_address"?

Reply:
According to the naming convention, the package where the models are to be declared  soud be named 'schemas'  You should remove the `schema` directrory and move the `schemas.py` file to the root (renamed from `schema.py`

Regarding these schemas, they should stay here.